airbyte-integrations/connectors/source-stripe/source_stripe/schemas/prices.json

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Prices Schema",
+  "additionalProperties": true,
+  "type": ["null", "object"],
+  "properties": {
+    "id": {
+      "type": ["null", "string"]
+    },
+    "object": {
+      "type": ["null", "string"]
+    },
+    "active": {
+      "type": ["null", "boolean"]
+    },
+    "billing_scheme": {
+      "type": ["null", "string"]
+    },
+    "created": {
+      "type": ["null", "number"]
+    },
```

Comment:
Looking at other streams, could this be a date-time formatted string instead of a number?